### DISCUSSION TITLE
.NET 7 TimeOnly support

### DISCUSSION THREAD

**Discussion by MichelMichels** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#2947] [Ideas]
Hi,

General inquiry if there's any interest in support the new `TimeOnly` struct for `TimePicker` control.

Thoughts?

**Comment by Keboo** [comment]
I certainly like the idea. Obviously, we will need to `#if` to limit the change to just NET 7 so that the other platforms continue compiling. I suspect it would likely be a breaking change as well so probably will want to get it in with the 5.x release.
